packages/design-system/src/components/OcSelect/OcSelect.vue

```diff
@@ -24,6 +24,35 @@
         <oc-spinner v-if="loading" />
       </template>
     </vue-select>
+
+    <div
+      v-if="showMessageLine"
+      class="oc-text-input-message"
+      :class="{
+        'oc-text-input-description': !!descriptionMessage,
+        'oc-text-input-warning': !!warningMessage,
+        'oc-text-input-danger': !!errorMessage
+      }"
+    >
+      <OcIcon
+        v-if="messageText !== null && !!descriptionMessage"
+        name="information"
+        size="small"
+        fill-type="line"
+        accessible-label="info"
```

Comment:
accessible label would need to be translated. Just leave it out (remove the attribute completely). As a result the OcIcon component will be aria-hidden. Which is ok, because it's only for visual improvement.